api-reference/user-app/vitals/vitals.yaml

```diff
@@ -0,0 +1,203 @@
+                    vital_loinc_code: 15074-8
+                    vital_value: 
+                      vital_numeric_value: 120
+                      vital_boolean_value: null
+                      vital_string_value: null
+                    unit_display_name: mg/dL
+                    unit_ucum_code: mg/dL
+                    measured_at:
+                      effectiveDateTime: '1693802400'
+                      instant: null
+                      period:
+                        startDateTime: null
+                        endDateTime: null
+      responses:
+        '200':
+          description: Vitals updated successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    example: true
+                  message:
+                    type: string
+                    example: Vitals updated successfully
+        '400':
+          description: Bad Request - Invalid data or missing parameters
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401':
+          description: Unauthorized - Invalid authentication token
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal Server Error
+      security:
+        - auth: []
+
+components:
+  schemas:
+    Vital:
+      type: object
+      properties:
+        vital_display_name:
+          type: string
+          description: Type of the vital sign (e.g., heart_rate, blood_pressure).
+        vital_loinc_code:
+          type: string
+          description: LOINC code corresponding to the vital sign.
+        vital_value:
+          type: object
+          description: Measured value of the vital sign. Only one type of vital_value is expected (vital_numeric_value or vital_boolean_value or vital_string_value)
+          properties:
+            vital_numeric_value:
+              type: number
+              description: Numeric value of the vital sign.
+            vital_boolean_value:
+              type: boolean
+              description: Boolean value of the vital sign, if applicable.
+            vital_string_value:
+              type: string
+              description: String value of the vital sign, if applicable.
+        unit_display_name:
+          type: string
+          description: Unit of the measured value (e.g., mmHg, mg/dL).
+        unit_ucum_code:
+          type: string
+          description: Unit code as per standard (e.g., {Counts}/min, mg/dL).
+        measured_at:
+          type: object
+          properties:
+            effectiveDateTime:
+              type: string
+              format: date-time
+              description: Date and time when the vital was measured.
```

Comment:
explain that it can be year/ year+month and such combinations, borrow explanation from FHIR website
https://build.fhir.org/datatypes.html#dateTime